"""tentando adicionar coluna de usuario.

Revision ID: 05235795c1ba
Revises: 
Create Date: 2022-06-29 13:57:06.282967

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = '05235795c1ba'
down_revision = None
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_table('user_control')
    op.add_column('gastos', sa.Column('UserControl_id', sa.Integer(), nullable=True))
    op.create_foreign_key(None, 'gastos', 'usuarios_control', ['UserControl_id'], ['id'])
    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_constraint(None, 'gastos', type_='foreignkey')
    op.drop_column('gastos', 'UserControl_id')
    op.create_table('user_control',
    sa.Column('id', sa.INTEGER(), autoincrement=True, nullable=False),
    sa.Column('username', sa.VARCHAR(length=20), autoincrement=False, nullable=True),
    sa.Column('nome', sa.VARCHAR(length=80), autoincrement=False, nullable=True),
    sa.Column('senha', sa.VARCHAR(length=14), autoincrement=False, nullable=True),
    sa.PrimaryKeyConstraint('id', name='user_control_pkey')
    )
    # ### end Alembic commands ###
